manifests/0000_90_cluster-svcat-apiserver-operator_02-operator-servicemonitor.yaml

```
@@ -25,3 +25,21 @@ spec:
   selector:
     matchLabels:
       app: openshift-service-catalog-apiserver-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: openshift-service-catalog-apiserver-operator
+  namespace: openshift-service-catalog-apiserver-operator
+spec:
+  groups:
+    - name: svcat-apiserver-enabled
+      rules:
+        - alert: ServiceCatalogAPIServerEnabled
+          annotations:
+            summary: Indicates whether Service Catalog is enabled
```

Nit: we're you thinking of changing this one as well?
```suggestion
            summary: Indicates whether Service Catalog API Server is enabled
```

---

Yep just did. Forgot about that.